Portfolio setup

anbehindY opened the pull request:
I added an index.html and styles.css that makes up the main page of the mobile version

Gambit142 reviewed (commented):
Hi @anbehindY 👋, 

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request

- Kindly use the recommended Microverse's README.md template https://github.com/microverseinc/readme-template. Please provide relevant information for all sections of the README.md file. The FAQ section is the only optional section. 👍 
- Kindly make your pull request summary descriptive. Your summary should describe the features you implemented in this branch 👍  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,38 @@
+<body>
+    <div class="wrapper">
+        <header>
+            <div id="name"><a href="#">John Doe</a></div>
```
- [ ] You used the correct anchor tag for the logo, but the color of the logo does not match with the template. The `a` tag is a little bit tricky when using parent styles, so I recommend you use the `color: inherit` attribute or add the color directly to the logo `a` tag.
 - Your logo:
 - 
<img width="324" alt="image" src="https://user-images.githubusercontent.com/61919617/219476700-80a446eb-a9cc-416a-aee0-b7329db70a26.png">

 - Template:
 - 
<img width="359" alt="image" src="https://user-images.githubusercontent.com/61919617/219476865-344a67f5-d2fa-4811-96b5-e96c4fc57c05.png">

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,38 @@
+            </nav>
+        </header>
+        <main>
+            <article class="headline-section">
+                    <h2>Hey There.<br>I’m Alex</h2>
+                    <h4>I’am a Software Developer</h4>
+                    <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.</p>
+                    <div class="socials">
+                        <a class="social" href=""><img src="icons/github.svg" /></a>
+                        <a class="social" href=""><img src="icons/linkedin.svg" /></a>
+                        <a class="social" href=""><img src="icons/angelist.svg" /></a>
+                        <a class="social" href=""><img src="icons/twitter.svg" /></a>
+                        <a class="social" href=""><img src="icons/medium.svg" /></a>
+                    </div>
+            </article>
```
- [ ] This is the first section of your page and a `section` tag would fit better than an `article` tag. Kindly change that for better HTML semantics.

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,38 @@
+        </header>
+        <main>
+            <article class="headline-section">
+                    <h2>Hey There.<br>I’m Alex</h2>
+                    <h4>I’am a Software Developer</h4>
```
- [ ] The main title("Hey There. I’m Alex") should be an `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly change that.

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @anbehindY 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️
 - Professional README file. ✔️ 
 - Correct use of GitHub flow. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

anbehindY commented:
I added index.html and styles.css files that make up the first page of the mobile version of the portfolio.
README.md is also updated.
All the linters are checked and green.
